components/Players/PlayerManager.vue

```diff
@@ -388,6 +390,25 @@ export default Vue.extend({
         }
       }
     },
+    addMediaHandlers(): void {
+      if (navigator.mediaSession) {
+        navigator.mediaSession.setActionHandler('previoustrack', () => {
+          this.setPreviousTrack();
+        });
+        navigator.mediaSession.setActionHandler('nexttrack', () => {
+          this.setNextTrack();
+        });
+        navigator.mediaSession.setActionHandler('play', () => {
+          this.playPause();
```

Comment:
Maybe we should leave this with the play action alone? Just in case there's some strange media session implementation. Non blocking though.

Reply:
Removed